cumulus/pallets/parachain-system/src/lib.rs

```diff
@@ -756,7 +757,10 @@ pub mod pallet {
 	pub(super) type DidSetValidationCode<T: Config> = StorageValue<_, bool, ValueQuery>;
 
 	/// The relay chain block number associated with the last parachain block.
+	///
+	/// This is updated in `on_finalize`.
 	#[pallet::storage]
+	#[pallet::getter(fn last_relay_block_number)]
```

Comment:
Given [this](https://github.com/paritytech/polkadot-sdk/issues/223), we probably should implement a dedicated function.